node/src/lib.rs

```diff
@@ -1,17 +1,48 @@
+//! A Simperby node.
+//!
+//! The following CLI commands are provided by `SimperbyNode` as they are based on the node state.
+//!
+//! - `sync`
+//! - `clean`
+//! - `vote`
+//! - `veto`
+//! - `consensus`
+//! - `git`
+//! - `show`
+//! - `network`
+//! - `update`
+//! - `broadcast`
+//! - `chat`
+//!
+//! The following CLI commands are provided as global functions as they are node-stateless.
```

Comment:
```suggestion
//! The following CLI command is provided as a global function as it is not based on the node state.
```

Reply:
I think it's better to keep it in plural form as it *happens to be** singular. There could be more commands in the future.